ci/ray_ci/test_anyscale_docker_container.py

```diff
@@ -18,15 +19,22 @@ def _mock_run_script(input: List[str]) -> None:
         with mock.patch(
             "ci.ray_ci.docker_container.Container.run_script",
             side_effect=_mock_run_script,
-        ):
-            container = AnyscaleDockerContainer("3.8", "cu11.8.0", "ray")
+        ), mock.patch.dict(os.environ, {"BUILDKITE_BRANCH": "test_branch"}):
+            container = AnyscaleDockerContainer("3.9", "cu11.8.0", "ray-ml")
```

Comment:
why do you need to change the python code to 3.9?

Reply:
python 3.8 has way more aliases because it's the default python version, but thats logic is irrelevant to this test case